### PR TITLE
[KYUUBI #2540] Kyuubi Spark TPC-DS Connector - SupportsNamespaces

### LINKED ISSUE #2540: [Subtask] Kyuubi Spark TPC-DS Connector - SupportsNamespaces
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues) and found no similar issues.

### Describe the subtask

`TPCDSCatalog` should implement `SupportsNamespaces` to support `SHOW DATABASES` command.

### Parent issue

https://github.com/apache/incubator-kyuubi/issues/2538

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by jiaoqingbo**
### _Why are the changes needed?_

fix #2540 

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSCatalog.scala [hunk 1]
```diff
@@ -72,4 +73,37 @@ class TPCDSCatalog extends TableCatalog {
...
+        Array()
+      case _ =>
+        throw new NoSuchNamespaceException(namespace)
+    }
+  }
+
+  override def loadNamespaceMetadata(namespace: Array[String]): util.Map[String, String] = {
+    namespace match {
+      case Array(_) =>
+        mutable.HashMap[String, String]().asJava
```
Comment (pan3793) [inline extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSCatalog.scala:95]: why mutable?

**Comment by pan3793** [comment]
Please add a test

File: extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSCatalog.scala [hunk 2]
```diff
@@ -20,16 +20,17 @@ package org.apache.kyuubi.spark.connector.tpcds
...
 
 import io.trino.tpcds.Table
 import org.apache.spark.sql.catalyst.analysis.{NoSuchNamespaceException, NoSuchTableException}
-import org.apache.spark.sql.connector.catalog.{Identifier, Table => SparkTable, TableCatalog, TableChange}
+import org.apache.spark.sql.connector.catalog.{Identifier, NamespaceChange, SupportsNamespaces, Table => SparkTable, TableCatalog, TableChange}
 import org.apache.spark.sql.connector.expressions.Transform
 import org.apache.spark.sql.types.StructType
 import org.apache.spark.sql.util.CaseInsensitiveStringMap
 
 // TODO: implement SupportsNamespaces
```
[the remark this answers is not in the document]
Comment (jiaoqingbo) [reply extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSCatalog.scala:32]: OK

File: extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSCatalog.scala (hunk 1 above)
Comment (jiaoqingbo) [reply extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSCatalog.scala:95]: I will modify it

**Comment by jiaoqingbo** [comment]
> Please add a test

ok

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merging to master

**Pull request closed without merging**